представления/вован-зт/выскакивать/index.html

```diff
@@ -0,0 +1,72 @@
+                            <li class="modal__item"><a href="#"> <img src="icons/chat.png" alt=""><span>item4</span></a></li>
+                            <li class="modal__item"><a href="#"> <img src="icons/classroom.png" alt=""><span>item5</span></a></li>
+                            <li class="modal__item"><a href="#"> <img src="icons/collection.png" alt=""><span>item6</span></a></li>
+                            <li class="modal__item"><a href="#"> <img src="icons/contacts.png" alt=""><span>item7</span></a></li>
+                            <li class="modal__item"><a href="#"> <img src="icons/documents.png" alt=""><span>item8</span></a></li>
+                            <li class="modal__item"><a href="#"> <img src="icons/files.png" alt=""><span>item9</span></a></li>
+                            <li class="modal__item"><a href="#"> <img src="icons/mail.png" alt=""><span>item10</span></a></li>
+                            <li class="modal__item"><a href="#"> <img src="icons/maps.png" alt=""><span>item11</span></a></li>
+                            <li class="modal__item"><a href="#"> <img src="icons/mars.png" alt=""><span>item12</span></a></li>
+                            
+                            <input  class="navigation__check_input" type="checkbox" name="" id="modal2">
+                            <label  for="modal2" class="navigation__image2"></label> 
+        
+                            <li class="modal__item modal__item2" ><a href="#"><img src="icons/collection.png" alt=""><span>item13</span></a></li>
+                            <li class="modal__item modal__item2" ><a href="#"><img src="icons/contacts.png" alt=""><span>item14</span></a></li>
+                            <li class="modal__item modal__item2" ><a href="#"><img src="icons/documents.png" alt=""><span>item15</span></a></li>
+                            <li class="modal__item modal__item2" ><a href="#"><img src="icons/files.png" alt=""><span>item16</span></a></li>
+                            <li class="modal__item modal__item2" ><a href="#"><img src="icons/mail.png" alt=""><span>item17</span></a></li>
+                            <li class="modal__item modal__item2" ><a href="#"><img src="icons/maps.png" alt=""><span>item18</span></a></li>
+                            <li class="modal__item modal__item2" ><a href="#"><img src="icons/mars.png" alt=""><span>item19</span></a></li>
+                            <li class="modal__item modal__item2" ><a href="#"><img src="icons/maps.png" alt=""><span>item20</span></a></li>
+                            <li class="modal__item modal__item2" ><a href="#"><img src="icons/mars.png" alt=""><span>item21</span></a></li>
+                        </ul>
+                    </div> 
+                </li>
+                <li  class="navigation__item"> 
+                    <input class="navigation__check_input" type="checkbox" >
+                    <label><img class="navigation__image" src="icons/bell.png" alt="bell3"></label> 
```

Comment:
If you remove `alt` value for icons in the popup why do you leave it for navigation?